src/Helper/Navigation/Breadcrumbs.php

```diff
@@ -111,9 +111,13 @@ public function renderStraight($container = null)
         } else {
             /** @var View\Helper\EscapeHtml $escaper */
             $escaper = $this->view->plugin('escapeHtml');
-            $html    = $escaper(
+            $label   = $escaper(
```

Comment:
The type seems valid here: re-generating the Psalm baseline should suffice. Not sure why `mixed` is inferred, but your code is good as-is :+1: